# A 2d version of a path.
std_msgs/Header header

geometry_msgs/Pose2D[] poses